### PR TITLE
[PATCH v1] linux-gen: ishm: optimize huge page usage

### PR DESCRIPTION
**Pull request by matiaselo**

### CODE REVIEW

**Comment by matiaselo** [comment]
There seems to be a random traffic manager failure on Travis. @muvarov could you please restart the test.

**Comment by muvarov** [comment]
@matiaselo  looks like all tests passed now.

**Comment by matiaselo** [comment]
> looks like all tests passed now.

Good, thanks!

File: platform/linux-generic/odp_ishm.c [hunk 1]
```diff
@@ -125,6 +125,13 @@
  */
 #define ISHM_NB_FRAGMNTS (ISHM_MAX_NB_BLOCKS * 2 + 1)
 
+/*
+ * Memory reservations larger than ISHM_HUGE_PAGE_LIMIT (bytes) are allocated
+ * using huge pages (if available). Smaller reservations are done using normal
+ * pages to conserve memory.
+ */
+#define ISHM_HUGE_PAGE_LIMIT (64 * 1024)
```
Comment (Bill-Fischofer-Linaro) [inline platform/linux-generic/odp_ishm.c]: Might be worth moving this to the config file at some point? I can see this being a tunable parameter.

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**